credmark/cmf/types/ledger.py

```diff
@@ -23,6 +24,9 @@ class LedgerModelOutput(IterableListGenericDTO[dict]):
         default=[], description='A list of dicts which are the rows of data')
     _iterator: str = PrivateAttr("data")
 
+    def to_df(self):
```

Comment:
can we call this to_dataframe(self)? So that we don't confuse people who don't know pandas?

Reply:
yes. changed to `to_dataframe()`